proxy.go

```diff
@@ -110,10 +111,39 @@ func main() {
 		Timeout: time.Second * 10,
 	}
 
-	http.HandleFunc(settingsString[LISTEN_PATH], uploadFile)
+	http.HandleFunc("/", proxyHandler)
 	http.ListenAndServe(":6743", nil)
 }
 
+func proxyHandler(w http.ResponseWriter, r *http.Request) {
+	if r.URL.Path == settingsString[LISTEN_PATH] && strings.HasPrefix(r.Header.Get("Content-Type"), "multipart/form-data") {
+		// Process request with a multipart/form-data
+		uploadFile(w, r)
+	} else {
+		// Forward the rest of requests
+		fmt.Println("Forwarding", r.Method, "request on", r.URL.Path)
+		forwardRequest(w, r)
+	}
+}
+
+func forwardRequest(w http.ResponseWriter, r *http.Request) {
+	parsedURL, _ := url.Parse(settingsString[FORWARD_DESTINATION])
+	baseURL := &url.URL{ // Get only the base URL to forward the whole request there
+		Scheme: parsedURL.Scheme,
+		Host:   parsedURL.Host,
+	}
+	proxy := httputil.NewSingleHostReverseProxy(baseURL)
```

Comment:
This only forwards to the base URL, not the path specified in the environment variable, isnt it?